otter/integration/tests/test_scaling.py

```diff
@@ -202,9 +202,23 @@ def test_policy_execution_after_adding_clb(self):
         """
         rcs = TestResources()
 
-        self.clb1 = cloud_load_balancer.CloudLoadBalancer(pool=self.pool)
+        def create_1st_load_balancer(self):
```

Comment:
This probably doesn't need to take "self", since it's closed over by the instance method.